nvram_plugin.py

```diff
@@ -0,0 +1,181 @@
+
+from pandare import PyPlugin, Panda
+import ipaddress
+import socket
+import os
+'''
+Interface to LibNvram Hypercalls.
+
+Register callbacks via 
+panda.pyplugins.ppp.NVramHC.ppp_reg_cb("NVRAM_GET",get_ipdb_call)
+For NVRAM_GET exmaple
+TODO: Setup actual list add. Handle additional nvram hooks
+'''
+class NVramHC(PyPlugin):
+    def __init__(self, panda: Panda):
+        print("Loading Nvram HyperCall plugin")
+        super().__init__(panda)
+        self.panda = panda
+        outdir = self.get_arg("outdir")
+        self.write_file = os.path.join(outdir,"nvram.log")
+        open(self.write_file, "w").close() #Clear the file
+        self.write2_file = os.path.join(outdir,"nvram2.log")
```

Comment:
Why 2 log files?

Reply:
one logs commands. one logs state. just need to develop a better state save.